tst/testbugfix/00088.tst

```diff
@@ -0,0 +1,5 @@
+# 2005/08/19 (JS)
+gap> f:=function() Assert(0,false); end;; g:=function() f(); end;;
+gap> ##  The following should just trigger a normal error, but in 4.4.5
+gap> ##  it will send a few hundred lines before crashing:
+gap> # g();
```

Comment:
This is a no-test, as `g()` is commented out... Shall we remove the comment? (again, it's OK by me if we do this after the PR is merged... I am just not sure what exactly the plan is, so I am adding comments as I look at this)